apiary.apib

```diff
@@ -4371,14 +4371,10 @@ This request is [asynchronous](#introduction/synchronous-and-asynchronous-calls)
         + Default: eq
     + whereFilters[] (optional)
         + column (required) - column which should be matched
-        + operator (required, enum[string]) - operator for value comparison
+        + operator (optional, enum[string]) - operator for value comparison
             + Members
                 + eq - Equals - can be used with multiple values
```

Comment:
ten `eq` tam nema zmysel posielat je to defaultne `eq` ale nechal som to tu nic sa nestane ked sa to posle